setup.py

```diff
@@ -19,7 +19,6 @@
     "coveralls",
     "mock",
     "pytz",
-    "django-filter<2;python_version<'3'",
     "django-filter>=2;python_version>='3'",
```

Comment:
can this be unconditional now, since we are python 3 only?

Reply:
Done.